Contents/Code/plex_api_helper.py

```diff
@@ -38,7 +38,7 @@
 q = queue.Queue()
 
 
-plex_url = 'http://localhost:32400'
+plex_url = 'http://127.0.0.1:32400'  # the explicit IPv4 address is used because `localhost` can resolve to ::1, which `websocket` rejects
```

Comment:
line too long, 120 is the limit

I kind of think it would be cleaner to separate this to a separate PR as well.